plasmapy/classes/tests/test_Plasma.py

```diff
@@ -95,3 +95,93 @@ def test_Plasma_derived_vars():
     assert test_plasma.alfven_speed.shape == test_plasma.density.shape
     assert test_plasma.alfven_speed.unit.si == u.m / u.s
     assert np.allclose(test_plasma.alfven_speed.value, 10.92548431)
+
+
+class Test_PlasmaBlob(object):
```

Comment:
You really like your Python 2 old style objects classes :D You don't need to inherit from `object` in 3.

Reply:
It's funny because I basically never learned Python 2.... It's just there's so much damn legacy code examples out there!